lib/jekyll/excerpt.rb

```diff
@@ -169,12 +177,11 @@ def liquid_block?(tag_name)
 
     def print_build_warning
       Jekyll.logger.warn "Warning:", "Excerpt modified in #{doc.relative_path}!"
-      Jekyll.logger.warn "",
-                         "Found a Liquid block containing separator '#{doc.excerpt_separator}'" \
-                         " and has been modified with the appropriate closing tag."
-      Jekyll.logger.warn "",
-                         "Feel free to define a custom excerpt or excerpt_separator in the" \
-                         " document's Front Matter if the generated excerpt is unsatisfactory."
+      Jekyll.logger.warn "", "Found a Liquid block containing separator" \
+                         " #{doc.excerpt_separator.inspect} "
+      Jekyll.logger.warn "", "and has been modified with the appropriate closing tag."
```

Comment:
This line here reads a bit funky. Could you change the warning to say something like the code block below?
```
Found a Liquid block containing the excerpt separator '#{doc.excerpt_separator.inspect}'. The block has been modified with the appropriate closing tag.
```

Reply:
Sure. Not a problem..